assets/src/edit-story/components/library/panes/media/mediaPane.js

```diff
@@ -312,6 +323,59 @@ function MediaPane(props) {
     [hasMore, isMediaLoading, isMediaLoaded, setNextPage]
   );
 
+  // Arranges elements in rows.
+  const rowBasedGallery = (
```

Comment:
If isRowBasedGallery is disabled, should this code be run, if 'rowBasedGallery' will be discarded?

Reply:
Refactored this a bit so it wouldn't create components that it's not planning to render anyway.